webrtc.html

```diff
@@ -1345,6 +1346,12 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is missing values for both <var>sdpMid</var> and
+                <var>sdpMLineIndex</var>, reject <var>p</var> with a <code>TypeError</code>
+                and jump to the step labeled <em>Return</em>.</p>
```

Comment:
Another formatting question: When do we say "this attribute MUST return x" as opposed to simply "this attribute returns x"? There seems to be some inconsistency.